pages_builder/push_to_github_pages.sh

```diff
@@ -0,0 +1,35 @@
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Worth setting the 'e' option with `set -e` to ensure the script exits if it hits any errors.